src/ServiceInsight/Saga/SagaWindowViewModel.cs

```diff
@@ -99,6 +106,11 @@ string GetMajorVersion(StoredMessage message)
             return version?.Split('.').First();
         }
 
+        void ClearSaga()
+        {
+            Data = null;
+        }
+
```

Comment:
Specially with this kind of code, it wouldn't be as clear why we're doing it when done directly int he event handler method.